backend/main/leaderboard_test.go

```diff
@@ -34,4 +32,71 @@ func TestGetCommunityLeaderboard(t *testing.T) {
 	assert.Equal(t, 2, len(p.Data))
 	assert.Equal(t, 3, p.Data[0].Score)
 	assert.Equal(t, 2, p.Data[1].Score)
```

Comment:
I wonder if we can be clear about what is being asserted here by storing these values in named variables?